src/test/java/be/seeseemelk/mockbukkit/entity/PlayerMockTest.java

```diff
@@ -1371,4 +1370,38 @@ public void testPlayerSpawnParticle_Incorrect_DataType()
 		});
 	}
 
+	@Test
+	public void testDisconnect()
```

Comment:
```suggestion
	void disconnect()
```

Reply:
We should probably establish Naming Conventions for Tests first